asset-monitor/src/autotask.js

```diff
@@ -0,0 +1,186 @@
+//  CTOKEN UNDERLYING ASSET MONITOR AGENT ALERT - note there's never been an alert to this agent yet
+// agent id - 0x3f02bee8b17edc945c5c1438015aede79225ac69c46e9cd6cff679bb71f35576
+
+const axios = require('axios');
+const needle = require('needle');
```

Comment:
Any reason that you are using `axios.post` and needle.post`? They both can do the same thing

Reply:
Also needle is missing from  the `package.json` > `dependencies` list